src/views/DefineCompany.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div>
+    <header>
+      <h2>1. Company Name</h2>
+    </header>
+
+    <v-card flat class="define-company-container">
+      <section>
+        Company Name component goes here.
+      </section>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Vue } from 'vue-property-decorator'
+
+@Component({})
```

Comment:
I know we were used to putting in the `({options})` but it isn't required. Is this how you would like to see the decorator done?

Reply:
Done like this. At one point I got lint errors when I didn't specify the `{}`.